nl_server/config.py

```diff
@@ -34,97 +35,84 @@
 NL_EMBEDDINGS_VERSION_KEY: str = 'NL_EMBEDDINGS_VERSION_MAP'
 
 
+class StoreType(str, Enum):
+  MEMORY = 'MEMORY'
+  LANCEDB = 'LANCEDB'
+
+
 # Defines one embeddings index config.
 @dataclass
 class EmbeddingsIndex:
   # Name provided in the yaml file, and set in `idx=` URL param.
   name: str
 
+  # Values are: MEMORY, LANCEDB
+  store_type: StoreType
+
   # File name provided in the yaml file.
-  embeddings_file_name: str
+  embeddings_path: str
   # Local path.
   embeddings_local_path: str
 
   # Fine-tuned model name ("" if embeddings uses base model).
-  tuned_model: str = ""
+  model_name: str = ""
   # Fine-tuned model local path.
-  tuned_model_local_path: str = ""
+  model_local_path: str = ""
 
 
 #
 # Validates the config input, downloads all the files and returns a list of Indexes to load.
 #
-def load(embeddings_map: Dict[str, str],
-         models_map: Dict[str, str]) -> List[EmbeddingsIndex]:
+def load(embeddings_map: Dict[str, Dict[str, str]]) -> List[EmbeddingsIndex]:
```

Comment:
nit: maybe comment on what the keys/values for the Dicts are or have a dataclass?

Reply:
The inputs are raw yaml representation being turned into dataclass in this function, so just added comments.